astropy/visualization/interval.py

```diff
@@ -203,9 +205,10 @@ class ZScaleInterval(BaseInterval):
 
     Parameters
     ----------
-    nsamples : int, optional
+    n_samples : int, optional
         The number of points in the array to sample for determining
-        scaling factors.  Defaults to 1000.
+        scaling factors.  Defaults to 1000.  (Name changed from ``nsamples``
+        in v5.2.)
```

Comment:
It would be better to use `.. versionchanged` here, see this example: https://github.com/astropy/astropy/blob/7308ca5326caf96f5230b66deecc0a96caa605fa/astropy/io/votable/table.py#L55-L68

Reply:
Done.